src/cpp/ext/otel/otel_server_call_tracer.cc

```diff
@@ -137,38 +131,27 @@ class OpenTelemetryServerCallTracer : public grpc_core::ServerCallTracer {
  private:
   absl::Time start_time_;
   absl::Duration elapsed_time_;
-  std::vector<std::pair<absl::string_view,
-                        absl::variant<absl::string_view, std::string>>>
-      labels_;
-  bool received_peer_labels_ = false;
+  std::string method_;
```

Comment:
Same question as on client side: Can we store this as a slice to avoid an allocation?

Reply:
done